Export.as

```diff
@@ -29,6 +29,22 @@ namespace VehicleState
 	//  3 = Rear Right
 	import float GetWheelDirt(CSceneVehicleVisState@ vis, int w) from "VehicleState";
 
+	// Get wheel falling state, and if in water. For w, use one of the following:
+	//  0 = Front Left
+	//  1 = Front Right
+	//  2 = Rear Left
+	//  3 = Rear Right
+	// The value returned seems to always be even (0, 2, 4, 6), but this could break.
```

Comment:
Can you change this line to say "This might be completely incorrect and give unexpected results"